wordle-web/src/views/AboutView.vue

```diff
@@ -1,19 +1,36 @@
 <template>
-  <v-sheet color="blue" height="170px" class="pt-10 my-5">
-    <v-card color="indigo-darken-3" class="ma-5" elevation="5" rounded="xl">
-      <v-card-title>Hello World</v-card-title>
-    </v-card>
-  </v-sheet>
+  <v-card class="mx-auto mt-12" max-width="844">
+    <v-img
+      src="https://www.wric.com/wp-content/uploads/sites/74/2023/04/patches.jpg?w=540"
```

Comment:
That is a chonky cat
![image](https://user-images.githubusercontent.com/43832670/236701297-da5f6f76-1063-41b2-9fd6-869d43aba522.png)